doc/manual/source/advanced-topics/distributed-builds.md

```diff
@@ -107,4 +107,18 @@ file included in `builders` via the syntax `@/path/to/file`. For example,
 causes the list of machines in `/etc/nix/machines` to be included.
 (This is the default.)
 
-[Nix instance]: @docroot@/glossary.md#gloss-nix-instance
\ No newline at end of file
+[Nix instance]: @docroot@/glossary.md#gloss-nix-instance
+
+## Resource Management
+
+Adding `resource-management` to the `experimental-features` setting in `nix.conf` enables a basic resource management scheme for system features. This is akin to what can be accomplished with job schedulers like Slurm, where a remote machine can have a limited quantity of a resource that can be temporarily "consumed" by a job. This can be used with memory-heavy builds, or derivations that require exclusive access to particular hardware resources.
+
+Resource management is supported in both the supported features and mandatory features of a remote machine configuration, by appending a colon `:` to a feature name followed by the quantity that this machine has. This is tracked on a per-store basis, so different users on a multi-user installation share the same pool of resources for their remote build machines. A derivation specifies that it consumes a resource with the same notation in the `requiredSystemFeatures` attribute.
```

Comment:
What does "this is tracked on a per-store basis" mean (which store)? The machine-that-wants-to-build-this-on-a-remote-builder's Nix store, or the machine-that-can-actually-build-this's Nix store? How is it tracked?

Reply:
Good point, clarified this at the end of the doc entry.